src/sidebar/templates/group-list-item-out-of-scope.html

```diff
@@ -0,0 +1,55 @@
+<div
+  class="group-list-item-out-of-scope__item"
+  ng-class="{'group-list-item__item': true, 'is-selected': false}"
+  ng-click="vm.toggleGroupDetails($event)"
+  tabindex="0"
+>
+  <!-- the group icon !-->
+  <div class="group-list-item__icon--container">
+    <img
+      class="group-list-item__icon group-list-item__icon--organization"
+      alt="{{ vm.orgName() }}"
+      ng-src="{{ vm.group.logo }}"
+      ng-if="vm.group.logo"
+    />
+  </div>
+  <!-- the group name and share link !-->
+  <div
+    ng-class="{'group-list-item-out-of-scope__details': true, expanded: vm.isDetailsExpanded}"
+  >
+    <svg
+      class="svg-icon group-list-item-out-of-scope__icon--unavailable"
+      xmlns="http://www.w3.org/2000/svg"
+      width="100%"
+      height="100%"
+      viewBox="0 0 24 24"
+    >
+      <path fill="none" d="M0 0h24v24H0V0z" />
+      <path
+        d="M11 15h2v2h-2zm0-8h2v6h-2zm.99-5C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
+      />
+    </svg>
+    <a
+      class="group-list-item-out-of-scope__name-link"
+      href=""
+      title="Group not annotatable on this domain."
+    >
+      {{vm.group.name}} </a
```

Comment:
The redness of the group name looks like a mistake to me, is it?

<img width="374" alt="out-of-scope-group-link" src="https://user-images.githubusercontent.com/2458/55018101-0cf78380-4fea-11e9-9681-d214257842ad.png">

I also think it is surprising here that the group name is a link, even though it doesn't visually look like one, whereas the "Why is this group unavailable?" message looks like a link but isn't.

Reply:
You can click anywhere on the group area to expand or select a group. I think the pointer does a nice job at indicating that when you hover over the space (although I realize that isn't a mobile accessible thing). I think the -Why is this group unavailable?- text is made to look more link like so the user knows the group is "clickable" even though the group is greyed out/visually not selectable. I'm not really that great at UI stuff so it's possible this could use some changes but it seems like something that should be done in a larger forum. All that being said I rather not make any changes to the styling of this PR since it's just a refactor. (The red name change was unintentional thank you for catching that!)

Reply:
OK. Let's just get the red-ness fixed and we can solicit feedback from others.